### DISCUSSION TITLE
Daily Journal: Importing the list component into the main module.

### DISCUSSION THREAD

**Discussion by dudley-codes** [nss-day-cohort-47/C47-Discussions#22] [Help Me] [answered]
**Name:** Alex Dudley

**Have you searched the other help tickets?**
There aren't any others at this time.

**Describe your issue**
I'm working on the last section of the Daily Journal. It's telling me to "import the list component into the main module" and that if I'm unclear how to do that I should look at the answer key repository for some guidance. There is no link for the answer key repository and I haven't been able to find anything helpfulon Google. 

**What have you googled?**
https://stackoverflow.com/questions/39601784/use-component-from-another-module

https://stackoverflow.com/questions/17433557/how-to-save-user-input-into-a-variable-in-html-and-js

https://www.w3schools.com/js/js_htmldom_collections.asp

**What have you tried?**
banging my head against my keyboard

**Comment by ardisrtilson** [comment, marked as answer]
Hi Alex!

In JournalEntry.js we define and **export** a function called EntryListComponent. This is a set of instructions which runs when we **invoke** the function. In this case the invocation would look like this: 

EntryListComponent()

When we **export** a function from a particular component, it is because we plan to use it in another component. In order to use it in the other component we must make an **import** statement in the component where we plan to use the function. In this case, the component where we import and invoke the EntryListComponent function is called main.js

**Comment by dudley-codes** [comment]
I missed a module and was overthinking things and thought my journal entries from the form were supposed to be populating in the DOM. 🤦‍♂️

**Reply by ardisrtilson** [reply]
It's better to overthink than underthink. Over time, you'll learn to think juuuust the right amount.
